realm/realm-library/src/main/java/io/realm/BaseRealm.java

```diff
@@ -152,14 +161,25 @@ public boolean isInTransaction() {
      */
     public void addChangeListener(RealmChangeListener listener) {
```

Comment:
Good point. We should probably add that to "best practices" on the website as well.